spec/support/redirect_expectation_helper.rb

```diff
@@ -36,16 +36,16 @@ def it_redirects_to_with_error_and_notice(path, error, notice)
     expect(flash[:caution]).to be_blank
   end
 
-  def it_redirects_to_with_comment_notice(path, notice)
```

Comment:
Some tests failed because they use the `it_redirects_to_with_comment_*` helpers. You should keep both the kudos and the comment versions of these helpers.